assets/src/components/field/color/Color.jsx

```diff
@@ -27,10 +27,16 @@ const Color = props =>{
    *
    * To avoid any issues, we use #FFFFFF as a default value
    */
+  let propertyValue
+  if ( typeof props.value === 'object' && Object.hasOwn( props.value, 'value' ) ) {
+    propertyValue = props.value.value === 0 ? '#FFFFFF' : props.value.value
+  } else {
+    propertyValue = props?.value ?? '#FFFFFF'

```

Comment:
This line behaves differently than the original condition, which compared the value to an empty string.

```js
props.value && props.value !== ''
```

The nullish coalescing operator `??` does not consider an empty string to be "nullish", so it could pass the value through as an invalid color. Might be nice to have a test against that if it's not too much trouble.

Reply:
Thanks, I have implemented the fixes in a718373, keeping logic consistent is important. Should I also commit the minified file to this PR?

Reply:
Yes, my thinking exactly, especially because for developers it might be better to use a development workflow with non-minified files and/or sourcemaps.